lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup_facts.py

```diff
@@ -171,6 +198,25 @@ def gather_dvs_portgroup_facts(self):
                         vlan_override=dvs_pg.config.policy.vlanOverrideAllowed
                     )
 
+                if self.params['show_vlan_info']:
```

Comment:
could you please isolate in a new function the code that you use to prepare `vlan_info`.

Reply:
Done.